tests/test_versioning_and_approval_http.py

```
@@ -162,6 +177,90 @@ async def create_test_document():
+        "Authorization": api_key,
+        "Content-Type": "application/json"
+    }
+
+    # First create a version to submit for approval
+    version_payload = {
+        "document_id": document_id,
+        "file_path": "/test/path/approval_test.pdf",
+        "change_description": "Test version for approval"
+    }
+
+    version_response = requests.post(
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)